library/old_protocol/old_protocol.cpp

```diff
@@ -0,0 +1,181 @@
+#include "old_protocol.hpp"
+
+#include <blackhole/attribute.hpp>
+
+#include "deserialize.hpp"
+#include "library/elliptics.h"
+#include "library/logger.hpp"
+
+namespace ioremap { namespace elliptics { namespace n2 {
+
+old_protocol_manager old_protocol_man; // Used globally since we currently have only one protocol
```

Comment:
Yes, it's only temporarily. Maybe we can resolve it in current PR, because we can. But current PR is already too heavy, and maybe it's good idea to put TODO and do it in some next PR. Let's wait shaitan opinion. I'll put TODO.

Reply:
We didn't come to an agreement. @antmat or @shaitan, we need your voices.

Reply:
Ok, will be done in separate commit.

Reply:
Done